extend.go

```diff
@@ -33,6 +33,13 @@ type Extender struct {
 	// Defaults to DefaultVariant. See the documentation of individual
 	// variants for more information.
 	Variant Variant
+
+	// Attributes are added to the <a> tag.
+	//
+	// Attributes are no escaped and must be valid attributes and values for HTML tags.
```

Comment:
This isn't an API I'd be comfortable exposing.
Proper handling of input is a pretty strong requirement.